packages/data-service/package.json

```diff
@@ -37,13 +37,15 @@
     "precompile": "npm run clean",
     "compile": "npm run compile-ts && gen-esm-wrapper . ./lib/.esm-wrapper.mjs",
     "compile-ts": "tsc -p tsconfig.json",
-    "cover": "nyc npm run test",
+    "cover": "nyc -x '**/*.spec.*' npm run test",
     "depcheck": "depcheck",
     "lint": "eslint . && prettier --check .",
     "bootstrap": "npm run compile",
     "prepublishOnly": "npm run compile",
-    "pretest": "npm run compile && mongodb-runner start --port=27018",
-    "test": "cross-env TS_NODE_FILES=true nyc mocha --colors --timeout 15000 -r ts-node/register src/**/*.spec.*",
+    "test": "cross-env TS_NODE_FILES=true mocha --colors --timeout 15000 -r ts-node/register **/*.spec.*",
+    "prepare": "npm run compile",
```

Comment:
We can't use `prepare` due to the https://github.com/mongodb-js/compass/pull/2356, having bootstrap and prepublishOnly should be enough for our usual monorepo flows or is there some other case that we don't cover with this?

```suggestion
```

Reply:
I don't really know where that comes from, i don't think i've added it, could be something that went wrong when i rebased